src/renderer/views/app/AppUpdateView.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { useMemo } from 'react'
+
+import * as RD from '@devexperts/remote-data-ts'
+import * as FP from 'fp-ts/function'
+
+import packageInfo from '../../../../package.json'
+import { AppUpdate, AppUpdateModalProps } from '../../components/AppUpdate'
+import { useAppUpdate } from '../../hooks/useAppUpdate'
+
+export const AppUpdateView: React.FC = () => {
+  const { appUpdater, resetAppUpdater } = useAppUpdate()
+
+  const updateModalProps = useMemo(
+    () =>
+      FP.pipe(
+        appUpdater,
+        RD.fold(
+          (): AppUpdateModalProps => ({ isOpen: false }),
+          (): AppUpdateModalProps => ({ isOpen: false }),
+          ({ message }): AppUpdateModalProps => ({ isOpen: true, type: 'fail', close: resetAppUpdater, message }),
+          (version): AppUpdateModalProps => ({
+            isOpen: true,
+            type: 'success',
+            goToUpdates: () => window.apiUrl.openExternal(`${packageInfo.repository.url}/releases/tag/v${version}`),
```

Comment:
[nitpick] Let's use `packageInfo.repository.url` in `ExternalUrl.GITHUB` (`src/shared/const.ts`). Or remove `ExternalUrl.GITHUB` there to use `packageInfo.repository.url` everywhere. _Just to avoid duplication ..._